src/main/webapp/style.css

```diff
@@ -0,0 +1,7 @@
+h3 {
+  text-align: center;
+}
+
+#results-box {
+  width: 50%;
+}
```

Comment:
Will this width styling be compatible with adding a RHS column on the page with a mock card?

Reply:
Maybe, but changed to a flexbox bc it'll definitely be easier to work with

Reply:
It's worth thinking early on about how you want to divide up the width. Is it percentage-based? Hardcoded pixel values? Will there be gaps? Dependent on screen size?

Flexbox definitely works here, but inline-block may be sufficient for your use case as well.

Reply:
Considering that we are displaying two main elements (search results and knowledge cards) the space will be shared 50/50. I want to proceed with flexbox for its (relative) simplicity, as I don't see any need to hardcode width values for now.

Reply:
Quick note - style.css is in head.tag, so it'll be included in both the homepage and the srp, but the .srp-container rule is specific to the srp. With the amount of CSS you have currently, it's not going to be an issue to have unused CSS on the homepage, but consider separating out common CSS and page-specific CSS.